### PR TITLE
Add team page to the website

### PR DESCRIPTION
**Pull request by cpfiffer**
Adds a team page to the website. This needs to be checked by all the team members for this is merged -- you need to go to the `docs/_data/members.yml` file and verify your personal information. I have left placeholders for the various bits of personal information that I did not know -- either delete the entry (if you don't have a twitter, for example) or correct it.

Also please check to make sure you are actually in the `members.yml` file if you are supposed to be. I am a forgetful person! 

You may add a photo of yourself in `assets/team_photos`, and then refer to the file by its name in the `members.yml` file. My bio is a good one to follow:

```yaml
- name: Cameron Pfiffer
  photo: cameron.jpg
  github: cpfiffer
  institution: University of Oregon
  website: cameron.pfiffer.org
  email: cpfiffer@uoregon.edu
  linkedin: cameron-pfiffer
```

The bios are in no particular order, so I'm open to suggests on how they should be ordered.

### CODE REVIEW

**Comment by devmotion** [comment]
I'm wondering if I'm supposed to be among the list of members? Should everyone in the TuringLang Github team be part of it?

**Comment by yebai** [comment]
> I'm wondering if I'm supposed to be among the list of members? Should everyone in the TuringLang Github team be part of it?

yes, David, you should be one of us. I think Cameron is only testing the setup so far, sorry for missing you out...

**Comment by devmotion** [comment]
> sorry for missing you out...

No worries :slightly_smiling_face: I'll add a section about me.

**Comment by cpfiffer** [comment]
Sorry @devmotion, I had your info up in my browser but I never actually copied it into the members page.

**Comment by cpfiffer** [comment]
And also sorry @sharanry. I have no idea what's going on, apparently.

**Comment by cpfiffer** [comment]
Alright, you now have to use a URL of some kind for your photo.

File: docs/_data/members.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  email: your-email@something.com
+  linkedin: linked-page
+  twitter: twitter-handle
+
+- name: Martin Trapp
+  photo: placeholder.png
+  github: trappmartin
+  institution: Graz University of Technology
+  website: https://martint.blog/
+  email: your-email@something.com
```
Comment (trappmartin) [inline docs/_data/members.yml]: martin.trapp@tugraz.at

File: docs/_data/members.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  photo: placeholder.png
+  github: xukai92
+  institution: University of Edinburgh
+  website: https://xuk.ai/
+  email: your-email@something.com
+  linkedin: linked-page
+  twitter: twitter-handle
+
+- name: Martin Trapp
+  photo: placeholder.png
```
Comment (trappmartin) [inline docs/_data/members.yml]: https://martintdotblog.files.wordpress.com/2018/06/photo.jpg

Do you scale all images?

**Comment by trappmartin** [comment]
Should we add @phipsgabler?

File: docs/_data/members.yml (hunk 2 above)
Comment (cpfiffer) [inline docs/_data/members.yml]:
```suggestion
  photo: https://martintdotblog.files.wordpress.com/2018/06/photo.jpg
```

Comment (cpfiffer) [reply docs/_data/members.yml]: Yeah, they're scaled.

File: docs/_data/members.yml (hunk 1 above)
Comment (cpfiffer) [inline docs/_data/members.yml]:
```suggestion
  email: martin.trapp@tugraz.at
```

**Pull request merged**